lib/dl_connector_bundle_chs3/dl_connector_bundle_chs3_tests/db/base/core/base.py

```diff
@@ -39,6 +39,7 @@
     create_s3_bucket,
     create_s3_client,
 )
+from dl_testing.utils import get_root_certificates
```

Comment:
unused import?

Reply:
oh, I expected `task cq:fix_dir -- ...` would fix it